### PR TITLE
e2e: fix agent installation

### PR DESCRIPTION
**Pull request by klihub**

### CODE REVIEW

File: test/e2e/run.sh [hunk 1]
```diff
@@ -479,7 +489,7 @@ launch() { # script API
         "cri-resmgr-agent")
             host-command "$SCP \"$adjustment_schema\" $VM_SSH_USER@$VM_IP:" ||
                 command-error "copying \"$adjustment_schema\" to VM failed"
-            vm-command "kubectl delete -f $(basename "$adjustment_schema"); kubectl create -f $(basename "$adjustment_schema")"
+            vm-command "kubectl delete -f $(basename "$adjustment_schema") || :; kubectl create -f $(basename "$adjustment_schema")"
```
Comment (askervin) [inline test/e2e/run.sh:492]: The latter kubectl is always executed after the former, regardless of the status. And as the latter kubectl alone sets the final exit status of the whole command, `|| :` has no effect here as far as I understand it.

**Comment by askervin** [comment]
Otherwise this looks good to me.

File: test/e2e/run.sh (hunk 1 above)
Comment (klihub) [reply test/e2e/run.sh:492]: The problem is, that without the || : the whole test aborts/fails after the 'kubectl delete ... $adjustment_schema' fails. At least that's what I think happened to me, but it's possible that I misdiagnosed the real cause.

Comment (askervin) [reply test/e2e/run.sh:492]: Just to make sure, tried this out by deleting `adjustment-schema.yaml` and then running ` launch cri-resmgr-agent`. It seems to work fine without `|| :`.

```
run.sh> launch cri-resmgr-agent
host> scp -o StrictHostKeyChecking=No -o ControlMaster=auto -o ControlPath=/home/ask/vms/data/n4c16-debian-sid-containerd/ssh -o ControlPersist=30 "/home/ask/go/src/github.com/intel/cri-resource-manage
r/pkg/apis/resmgr/v1alpha1/adjustment-schema.yaml" debian@172.17.0.2:
root@vm> kubectl delete -f adjustment-schema.yaml; kubectl create -f adjustment-schema.yaml
Error from server (NotFound): error when deleting "adjustment-schema.yaml": customresourcedefinitions.apiextensions.k8s.io "adjustments.criresmgr.intel.com" not found
customresourcedefinition.apiextensions.k8s.io/adjustments.criresmgr.intel.com created
root@vm> NODE_NAME=$(hostname) cri-resmgr-agent -kubeconfig /root/.kube/config  >cri-resmgr-agent.output.txt 2>&1 &
```

Comment (klihub) [reply test/e2e/run.sh:492]: Ok, thanks.

**Review by askervin — APPROVED** [review]
LGTM

**Pull request merged**